module-network.tf

```diff
@@ -131,54 +132,56 @@ output "nat_route_table_id" {
 }
 
 # Subnets
-output "subnet_ids" {
-  description = "Map of subnet ids by role for the cluster and associated resources."
-  value       = module.network.subnet_ids
-}
-output "subnet_cidrs" {
-  description = "Map of provided/calculated subnet CIDR ranges by role for the cluster."
-  value       = module.network.subnet_cidrs
-}
 output "bastion_subnet_id" {
-  value = lookup(module.network.subnet_ids, "bastion", null)
+  value = module.network.bastion_subnet_id
+}
+output "bastion_subnet_cidr" {
+  value = module.network.bastion_subnet_cidr
 }
 output "operator_subnet_id" {
-  value = lookup(module.network.subnet_ids, "operator", null)
+  value = module.network.operator_subnet_id
+}
+output "operator_subnet_cidr" {
+  value = module.network.operator_subnet_cidr
 }
 output "control_plane_subnet_id" {
-  value = lookup(module.network.subnet_ids, "cp", null)
+  value = module.network.control_plane_subnet_id
+}
+output "control_plane_subnet_cidr" {
+  value = module.network.control_plane_subnet_cidr
 }
 output "worker_subnet_id" {
-  value = lookup(module.network.subnet_ids, "workers", null)
+  value = module.network.worker_subnet_id
+}
+output "worker_subnet_cidr" {
+  value = module.network.worker_subnet_cidr
 }
 output "pod_subnet_id" {
-  value = lookup(module.network.subnet_ids, "pods", null)
+  value = module.network.pod_subnet_id
+}
+output "pod_subnet_cidr" {
+  value = module.network.pod_subnet_cidr
 }
 output "int_lb_subnet_id" {
-  value = lookup(module.network.subnet_ids, "int_lb", null)
+  value = module.network.int_lb_subnet_id
+}
+output "int_lb_subnet_cidr" {
+  value = module.network.int_lb_subnet_cidr
 }
 output "pub_lb_subnet_id" {
-  value = lookup(module.network.subnet_ids, "pub_lb", null)
+  value = module.network.pub_lb_subnet_id
+}
+output "pub_lb_subnet_cidr" {
+  value = module.network.pub_lb_subnet_cidr
 }
 output "fss_subnet_id" {
-  value = lookup(module.network.subnet_ids, "fss", null)
+  value = module.network.fss_subnet_id
 }
-
-# NSGs
-output "nsg_ids" {
-  description = "Map of network security group IDs by role for the cluster and associated resources."
-  value = var.create_nsgs ? {
-    "bastion"  = module.network.bastion_nsg_id
-    "operator" = module.network.operator_nsg_id
-    "cp"       = module.network.control_plane_nsg_id
-    "int_lb"   = module.network.int_lb_nsg_id
-    "pub_lb"   = module.network.pub_lb_nsg_id
-    "workers"  = module.network.worker_nsg_id
-    "pods"     = module.network.pod_nsg_id
-    "fss"      = module.network.fss_nsg_id
-  } : null
+output "fss_subnet_cidr" {
+  value = module.network.fss_subnet_cidr
 }
 
+# NSGs
```

Comment:
For `pod_nsg_id`, can you please clarify with the provider/service team whether this is supposed to be a single nsg or a list of nsgs? 

See: https://github.com/oracle/terraform-provider-oci/issues/1662

Reply:
Added a commit to address.